full-service/src/db/account.rs

```diff
@@ -492,16 +492,11 @@ impl AccountModel for Account {
 
         let mut accounts: Vec<Account> = Vec::<Account>::new();
 
-        if let Some(received_account_id_hex) = txo.received_account_id_hex {
+        if let Some(received_account_id_hex) = txo.account_id_hex {
```

Comment:
small nit: rename `received_account_id_hex` to just `account_id_hex`. If we're not differentiating between received and minted, I think that not using the word `received` will avoid confusion for future devs

Reply:
Hmm, I think that's the case currently. The txo table has a single account reference called "account_id_hex"

Reply:
ah I see! thanks, updated :)